src/main/java/org/jenkinsci/plugins/docker/traceability/fingerprint/DockerInspectImageFacet.java

```diff
@@ -68,7 +68,7 @@ public DockerInspectImageFacet(@Nonnull Fingerprint fingerprint, long timestamp,
     
     /**
      * Time, when the latest report has been submitted.
-     * @return The time is specified in milliseconds since January 1, 1970, 00:00:00 GMT
+     * @return The time is specified in seconds since January 1, 1970, 00:00:00 GMT
      */
     public long getReportTimestamp() {
```

Comment:
I think internally we should be using normal java timestamps, or we should use Dates.
Someone somehere will get a conversion wrong.